sdks/python/apache_beam/transforms/ptransform_test.py

```diff
@@ -2015,8 +2015,8 @@ def test_runtime_type_check_python_type_error(self):
     # Our special type-checking related TypeError shouldn't have been raised.
     # Instead the above pipeline should have triggered a regular Python runtime
     # TypeError.
-    self.assertEqual("object of type 'int' has no len() [while running 'Len']",
-                     e.exception.args[0])
+    expected_start = "object of type 'int' has no len() [while running 'Len']"
+    self.assertEqual(expected_start, e.exception.args[0][:len(expected_start)])
```

Comment:
can we use str.startswith() here?

Reply:
Will do. I didn't use `startswith` because I had confused it with `strip` (which matches characters, not strings).